### PR TITLE
Add Documentation 

### PR DESCRIPTION
**Pull request by Hbbb**
* Adds usage documentation
* Fixes `env file` command to use `appName` flag instead of taking it as an argument.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,85 @@
...
+## Commands
+
+## Managing Application Environment
+
+### `tuber env set`
+Set an environment variable
+
+```bash
+USAGE
+  $ tuber env set -a <app_name>
```
Comment (jkminneti) [inline README.md]: also takes key value

File: README.md [hunk 2]
```diff
@@ -1,9 +1,85 @@
-# tuber 
+# tuber
 
 ![logo](logo.png)
```
Comment (Jefferson-Faseler) [inline README.md]: Only reason I came to review the PR. 👍 

The logo is safe

File: README.md [hunk 3]
```diff
@@ -1,9 +1,85 @@
...
 
+## Install
+Download the binary file from the latest release: https://github.com/Freshly/tuber/releases/
+
+## What is Tuber?
+
+Tuber is primarily a command line tool used provide an ergonomic way to manage a cluster running in Google Kubernetes Engine. It provides commands for common operations such as running commands and getting/setting environment variables.
```
Comment (Jefferson-Faseler) [inline README.md]:
```suggestion
Tuber is primarily a command line tool used to provide an ergonomic way to manage a cluster running in Google Kubernetes Engine. It provides commands for common operations such as running commands and getting/setting environment variables.
```

File: cmd/env.go [hunk 4]
```diff
@@ -29,15 +29,15 @@ var envUnsetCmd = &cobra.Command{
 
 var fileCmd = &cobra.Command{
 	SilenceUsage: true,
-	Use:          "file [app] [local filepath]",
+	Use:          "file [local filepath]",
 	Short:        "batch env set",
-	Args:         cobra.ExactArgs(2),
+	Args:         cobra.ExactArgs(1),
```
Comment (Jefferson-Faseler) [inline cmd/env.go]: You're an inspiration to us all

**Pull request merged**